core/src/db/club/club.entity.ts

```diff
@@ -1,5 +1,8 @@
 import { BaseEntity } from '../base.entity';
-import { Entity } from 'typeorm';
+import { Column, Entity } from 'typeorm';
 
 @Entity('club', { schema: 'sprocket' })
-export class ClubEntity extends BaseEntity {}
+export class ClubEntity extends BaseEntity {
+	@Column()
+	franchiseId: string;
```

Comment:
This should either just be type Franchise, or have both Franchise and franchiseId. The important bit is the relation to the Franchise entity via TypeORM, in this case via OneToMany